DEVELOPMENT.md

```diff
@@ -85,8 +85,15 @@ export STAIRWAY_DATABASE_USER=stairwayuser
 export STAIRWAY_DATABASE_USER_PASSWORD=stairwaypwd
 export STAIRWAY_DATABASE_NAME=stairwaylib
 export SAMPLING_PROBABILITY=1.0
+export CLOUD_TRACE_SA_PATH=rendered/service-account.json
+export SERVICE_GOOGLE_PROJECT=broad-dsde-dev
+export SAM_ADDRESS=https://sam.dsde-dev.broadinstitute.org
+```
+
+and render configs if you haven't already 
+```sh
```

Comment:
Nit: add newline before code block (technically correct, some markdown readers require it)